src/Cardano/Fee.hs

```diff
@@ -445,18 +448,56 @@ remainingFee opts s = do
     Fee fee = estimateFee opts s
     diff = inputBalance s - (outputBalance s + changeBalance s)
 
--- Equally split the extra change obtained when picking new inputs across all
--- other change. Note that, it may create an extra change output if:
+-- | Splits up the given coin of value __@v@__, distributing its value over the
+--   given coin list of length __@n@__, so that each coin value is increased by
+--   the same absolute rational amount __@v/n@__ and then rounded to the nearest
+--   integer, producing a new list of coin values where the overall total is
+--   preserved.
+--
+-- == Examples
+--
+-- >>> splitCoin (Coin 4) (Coin <$> [1, 1, 1, 1])
+-- [Coin 2, Coin 2, Coin 2, Coin 2]
+--
+-- >>> splitCoin (Coin 4) (Coin <$> [1, 2, 3, 4])
+-- [Coin 2, Coin 3, Coin 4, Coin 5]
```

Comment:
Perhaps these examples could use a higher value for `coinToSplit`, to better illustrate that each coin is increased by an identical amount.

Reply:
Fixed in 6f5bd77559cb4d27d81ba71f2bee53dc761ada49.